src/form.js

```diff
@@ -138,6 +138,7 @@ define([
         this.errors = [];
         this.question_counter = 1;
         this.xpath = escapedHashtags.parser(this.hashtagDictionary);
+        this.noRichText = !vellum.opts().features.rich_text;
```

Comment:
Can we please change this to `this.richText = !!vellum.opts().features.rich_text;` and then change all of the places that say `!form.noRichText` to `form.richText`?